### DISCUSSION TITLE
Prestashop 1.7.8.11 : Does it support MariaDB 10.6 or 10.11

### DISCUSSION THREAD

**Discussion by githubjonny** [PrestaShop/PrestaShop#36065] [Q&A]
I've been unable to find online if Prestashop 1.7.8.11 supports MariaDB 10.6 or 10.11.

Anyone know the answer? (assume vanilla install / no 3rd party modules themes etc).

Official Prestashop docs don't seem to have it listed anywhere.

**Comment by kpodemski** [comment]
It supports both version since realistically speaking they are equivalent of MySQL.

There are some minor differences but they shouldn't impact PrestaShop 1.7.8. If I have to suggest something, I'd go for 10.6 because of some MySQL8-specific things that may not work in 1.7.8.11.

**Comment by matks** [comment]
See https://github.com/PrestaShop/PrestaShop/issues/33157#issuecomment-2090821496
